drivers/char/Kconfig

```diff
@@ -483,6 +483,48 @@ config RUST_EXAMPLE
 
 	  If unsure, say N.
 
+# XXX: Only for GitHub -- do not commit into mainline
```

Comment:
maybe you want to just put these in drivers/staging/rust/ and send them upstream?

Reply:
I think the "real" one should go eventually to `samples/`. The other 3 dummy copies are only for the GitHub CI, so if we want to keep them, perhaps they could go wherever we start putting Rust-related tests, e.g. in `rust/tests/`. Let me open an issue...

Reply:
https://github.com/Rust-for-Linux/linux/issues/65